docs/mobile/overview.mdx

```diff
@@ -15,75 +16,58 @@ Geocortex Mobile was designed to be highly configurable out of the box. Geocorte
+
+## Custom Code
 
-## Take your Application Further with the SDK
+<!-- TODO - update the component interactions link once the mobile restructure is merged in -->
 
-Use the [Geocortex Mobile SDK](sdk-getting-started.mdx) and [Geocortex Workflow SDK](../workflow/sdk-overview.mdx) to build completely custom UI and behavior.
+If you need to build entirely custom and new behavior and UI, you should check out the [**Geocortex Mobile SDK**](sdk-getting-started.mdx). Using the Geocortex Mobile SDK, you can develop custom behavior by creating new [commands and operations](usecases-implement-command-operation.mdx) which can be [used in the app config](concepts-commands-operations.mdx). If you need to create a new UI element, you can build [custom components](sdk-create-component.mdx). With the ability to create [background services](usecases-implement-service.mdx), facilitate [component interactions](https://github.com/geocortex/vertigis-mobile-samples/tree/master/Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/CustomSamples/BasemapToggle), and [participate in the app config](usecases-implement-component-participate-app-config.mdx), the possibilities for extending Geocortex Mobile are endless.
+
+## Next Steps
 
 <UseCaseContainer>
     <UseCaseCard
-        title="Implement a Custom Form Element with Geocortex Workflow"
-        description="Extend Geocortex Workflow forms with custom UI and functionality"
-        link="../workflow/usecases-mobile-implement-custom-form-elements"
-    />
-    <UseCaseCard
-        title="Create a Custom Activity with Geocortex Workflow"
-        description="Define custom behavior in Geocortex Workflow that can be leveraged in Geocortex Mobile"
-        link="../workflow/usecases-mobile-implement-custom-activity"
+        title="Layout and App Config"
+        description="Learn more about the layout and app config"
+        link="concepts-key-concepts"
     />
     <UseCaseCard
-        title="Extend Geocortex Mobile's Functionality with Custom Commands and Operations"
-        description="Add commands and operations to Geocortex Mobile's infrastructure that implement custom behavior"
-        link="usecases-implement-command-operation"
+        title="Running Workflows in Geocortex Mobile"
+        description="Learn how to build and run a workflow in Geocortex Mobile"
+        link="https://docs.geocortex.com/mobileviewer/latest/admin-help/Default.htm#gmv/designer/add-a-workflow-to-an-app.htmm"
     />
     <UseCaseCard
-        title="Create UI Elements with Custom Components"
-        description="Create fully custom UI components that can be used in Geocortex Mobile apps"
-        link="usecases-implement-component-with-ui"
+        title="Building Custom Workflow Activities"
+        description="Learn how to build custom workflow activities for Geocortex Mobile"
+        link="usecases-implement-custom-activity"
```

Comment:
I imagine this link doesn't work anymore after the rename?

Reply:
No it wouldn't. I merged in master and fixed all the links.